fluentd/configs.d/openshift/filter-syslog-record-transform.conf

```diff
@@ -37,6 +37,7 @@
     # Plus we'd better avoid the file IO.
     hostname ${_HOSTNAME.eql?('localhost') ? (begin; File.open('/etc/docker-hostname') { |f| f.readline }.rstrip; rescue; _HOSTNAME; end) : _HOSTNAME}
     message ${record["MESSAGE"]}
+    level ${record["PRIORITY"]}
```

Comment:
In the common data model, I believe we look for the syslog priority numbers to be changed to the names defined by syslog, while adding names that don't map.